runtime/file.cpp

```diff
@@ -13,7 +13,13 @@
 #include <cstring>
 #include <fcntl.h>
 #include <stdlib.h>
+#ifndef _WIN32
```

Comment:
Elsewhere in this code, you've organized the dependent variants with `ifdef _WIN32` and put the Windows code first.